spec/knapsack_pro/formatters/time_tracker_specs.rb

```diff
@@ -413,7 +413,7 @@ def test_subset
 
   def run_specs(specs)
     paths = Array(specs).map.with_index do |spec, i|
-      path = "spec/knapsack_pro/formatters/#{i}_#{SecureRandom.uuid}_spec.rb"
+      path = "spec/knapsack_pro/formatters/time_tracker_#{i}_#{SecureRandom.uuid}_spec.rb"
```

Comment:
You can use https://github.com/KnapsackPro/knapsack_pro-ruby/commit/5bbb2469df07f0705e530b84a543636f4b96b9b7 to avoid this and the gitignore.

Reply:
Applied.